docs/usage.md

```diff
@@ -22,29 +22,38 @@ An example is shown below.
 fullpipeline: true
 maponly: false
 cageronly: false
-gtf: "testdata/sacCer3_genome/sacCer3.ensGene.gtf"
+gtf: "cageflow_test_data/danRer11_genome/danRer11.ensGene.gtf"
 
 # preprocessing parameters
-input: "docs/examples/samplesheet_sacer_pe.csv"
-infolder:
-sample_name_fields:
-# mapping parameters
-genome_name: "sacCer3"
-fasta: "testdata/sacCer3_genome/sacCer3.fa"
-index: "testdata/sacCer3_genome/sacCer3_star_index/"
+samplesheet: "/mnt/biggley/home/slava/projects/leancage_dev/pe_samplesheet.csv"
+infolder: 
+outdir: "results"
+sample_name_fields: 3
+genome_name: "danRer11"
+genome: "cageflow_test_data/danRer11_genome/danRer11.fa"
+index: 
 seq_platform: "illumina"
 seq_center: false
 # whether to take the uniquely mapped only
 unique_only: true
 # whether to remove reads that do not start with G
-remove_non_g: false
+remove_non_g: true
+# whether to trim G or not
+nogtrim: false
+# mapper to use
+bowtie2: true
+# dedup
+dedup: true
+dist: 100
```

Comment:
i think this is not part of params, adjust schema and add with default parameters into the customcageq.nf file